language/diem-framework/tests/DiemCurrenciesTests.move

```diff
@@ -1,15 +1,18 @@
 // Check that the add_all_currencies flag does the expected thing
```

Comment:
Can you move this comment closer to `fun currencies`, since it only applies to that particular test, not the whole module?